doc/programming_guide/sound.rst

```diff
@@ -0,0 +1,724 @@
+#. :ref:`Platformer Tutorial - Step 7 - Add Coins And Sound <platformer_part_seven>`
+
+   #. :ref:`platformer_part_seven_loading_sounds`
+   #. :ref:`platformer_part_seven_playing_sounds`
+
+.. _sound-why-important:
+
+Why Is Sound Important?
+-----------------------
+
+Sound helps players make sense of what they see.
+
+This is about far more than the game being immersive or cool. For
+example, have you ever run into one of these common problems?
+
+* Danger you never knew was there
+* A character whose reaction seemed unexpected or out of place
+* Items or abilities which appeared similar, but were very different
+* An unclear warning or confirmation dialog
+
+How much progress did it cost you? A few minutes? The whole playthrough?
+More importantly, how did you feel? You probably didn't want to keep
+playing.
+
+You can use sound to prevent moments like these. In each example above,
+the right audio can provide the information players need for the game
+to feel fair.
+
+.. _sound-basics:
+
+Sound Basics
+------------
+
+.. _sound-basics-loading:
+
+Loading Sounds
+^^^^^^^^^^^^^^
+
+Before you can play a sound, you need to load its data into memory.
+
+Arcade provides two ways to do this. Both accept the same arguments and
+return an :py:class:`arcade.Sound` instance.
+
+The easiest way is :py:func:`arcade.load_sound`::
+
+    import arcade
+
+    # You can pass strings containing a built-in resource handle,
+    hurt_sound = arcade.load_sound(":resources:sounds/hurt1.wav")
+    # a pathlib.Path,
+    pathlib_sound = arcade.load_sound(Path("imaginary\windows\path\file.wav"))
+    # or an ordinary string describing a path.
+    string_path_sound = arcade.load_sound("imaginary/mac/style/path.wav")
+
+If you prefer a more object-oriented style, you can create
+:py:class:`~arcade.Sound` instances directly::
+
+    from arcade import Sound  # You can also use arcade.Sound directly
+
+    # Although Sound accepts the same arguments as load_sound,
+    # only the built-in resource handle is shown here.
+    hurt_sound = Sound(":resources:sounds/hurt1.wav")
+
+See the following to learn more:
+
+#. :ref:`Platformer Part 7 - Loading Sounds <platformer_part_seven_loading_sounds>`
+#. :ref:`resources`
+#. :py:mod:`pathlib`
+#. :ref:`sound-loading-modes`
+
+.. _sound-basics-playing:
+
+Playing Sounds
+^^^^^^^^^^^^^^
+
+There are two easy ways to play a :py:class:`~arcade.Sound` object.
+
+One is to call :py:meth:`Sound.play <arcade.Sound.play>` directly::
+
+    self.hurt_player = hurt_sound.play()
+
+The other is to pass a :py:class:`~arcade.Sound` instance as the first
+argument of :py:func:`arcade.play_sound`::
+
+    # Important: this *must* be a Sound instance, not a path or string!
+    self.hurt_player = arcade.play_sound(hurt_sound)
+
+Both return a :py:class:`pyglet.media.player.Player`. You should store
+it somewhere if you want to be able to stop or alter a specific playback of
+a :py:class:`~arcade.Sound`'s data.
+
+Sounds vs pyglet Players
```

Comment:
Confusing? Maybe:
The arcade sound class and pyglet players

Reply:
I've tried to fix it for now using alternate phrasing, but this is something which might improve upstream: there's discussion of renaming `pyglet.media.Player` to `pyglet.media.AudioPlayer`:

> I've thought about potentially renaming the AudioPlayer classes to something more generic, so it's not confusing with the high level Player.
> -Benjamin

[*Source: Discord message*](https://discord.com/channels/438622377094414346/1162528803373977681/1164140154328465458)

I'm not sure which version this could release in since there's an ongoing sound backend refactor / bug fix effort.